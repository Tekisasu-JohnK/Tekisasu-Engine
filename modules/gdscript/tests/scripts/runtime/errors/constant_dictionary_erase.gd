const dictionary := {}

func test():
	dictionary.erase(0)
