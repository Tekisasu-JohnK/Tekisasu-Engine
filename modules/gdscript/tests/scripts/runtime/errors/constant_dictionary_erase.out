GDTEST_RUNTIME_ERROR
>> ERROR
>> on function: erase()
>> core/variant/dictionary.cpp
>> 177
>> Condition "_p->read_only" is true. Returning: false
>> Dictionary is in read-only state.
